Framework/Core/include/Framework/TableTreeHelpers.h

```diff
@@ -122,8 +122,12 @@ class TreeToTable
  private:
   std::shared_ptr<arrow::Table> mTable;
   std::vector<std::string> mColumnNames;
+  std::string mTableLabel;
```

Comment:
Why an extra mTableLabel here? Don't you use the metadata of mTable?

Reply:
Since this class creates the table, I need to supply it with label first.